net/wireguard/files/wireguard.sh

```diff
@@ -161,6 +140,13 @@ proto_wireguard_setup() {
     exit 1
   fi
 
+  # endpoint dependency
+  wg show ${config} endpoints | while IFS=$'\t:' read -r key ip port; do
```

Comment:
Elsewhere in the script you quote ${config}, but not here. I have no idea if quoting of that variable is required, but I did notice the inconsistency. Probably doesn't matter, but FYI.